vfcompositor/app/GUIShell.c

```
@@ -0,0 +1,47 @@
+#include "Scheduler.h"
+
+static char command_buffer[256];
+static int cmd_pos = 0;
+extern char current_dir[256];
+
+extern CompositorContext g_compositor_ctx;
+extern void ConsoleSetWindowPrint(Window* w);
+
+void VFShellProcess(void) {
+    Window* w = CreateWindow(&g_compositor_ctx, 0, 0, 480, 360, "VFShell", GetCurrentProcess()->pid);
+    if (!w) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)